packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -92,8 +95,10 @@ class MethodWebsocketRequestHandler {
         }
       }
     } catch (e, stackTrace) {
-      var session = await server.serverpod.createSession();
-      await session.close(error: e, stackTrace: stackTrace);
+      stderr
```

Comment:
Will this not make the server noisy if a faulty client sends messages to it?

Reply:
That makes sense! I stole this logging approach from the current streaming implementation: https://github.com/serverpod/serverpod/blob/f5983a2bfb1578003274367d504ab87056f1355f/packages/serverpod/lib/src/server/websocket_request_handlers/endpoint_websocket_request_handler.dart#L136-L140

But I will add a safeguard only logging these issues if we have verbose (or some other suitable log setting) logging enabled.

Reply:
Added a guard to check if logging malformed calls is enabled. Do you think that is a good solution?

Reply:
That sounds like the preferred way to do it! 😀